src/app/patient-list-cohort/cohort-member/add-cohort-member.component.ts

```diff
@@ -77,6 +77,12 @@ export class AddCohortMemberComponent implements OnInit {
 
     }
 
+    getSelectedPatient(patient) {
```

Comment:
since this is an event handler, should be renamed to 'patientSelected'. Look at https://angular.io/guide/styleguide